tests/src/python/test_qgsanalysis.py

```diff
@@ -45,7 +36,7 @@ def tearDown(self):
         """Run after each test."""
         pass
 
-    #@expectedFailure
+    #@unittest.expectedFailure
```

Comment:
I remember that @timlinux told me once that if they succeed despite being marked expectedFailure they should trigger an error so you should notice.